config/scorecard/patches/olm.config.yaml

```diff
@@ -0,0 +1,50 @@
+- op: add
```

Comment:
does scorecard give us anything upfront? can we add it separately if/when we buy in?